client/src/views/page-components/account/AccountHistoryModal/AccountHistoryModal.scss

```
@@ -0,0 +1,215 @@
+@keyframes slidein {
+  from {
+    transform: translateX(600px);
+    opacity: 0;
+  }
+
+  to {
+    transform: translateX(0px);
+    opacity: 1;
+  }
+}
+
+@keyframes slideOut {
+  from {
+    transform: translateX(0px);
+    opacity: 1;
+  }
+  to {
+    transform: translateX(600px);
+    opacity: 0;
+  }
+}
+
+.account-history-modal {
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.25);
+  color: var(--concept-color-span);
+  font-size: 30px;
+  font-weight: bold;
+}
+
+.overlay {
+  width: 100%;
+  height: 100%;
+}
+
+.history-form {
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 600px;
+  height: 100%;
+  padding: 50px;
+  background: #f4f6fc;
+  border-radius: 50px 0px 0px 0px;
+  animation-duration: 2s;
+  animation-name: slidein;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &.hide {
+    animation-duration: 2s;
+    animation-name: slideOut;
+  }
```

요렇게 하면 keyframes 지우셔도 될 것 같습니다!

```suggestion
  display: flex;
  flex-direction: column;
  align-items: center;
  transform: translateX(600px);
  opacity: 0;
  transition: all .5s ease-in-out;

  &.hide {
    transform: translateX(0px);
    opacity: 1;
  }
```